frontend/src/pages/modelServing/screens/metrics/SPDChart.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { InferenceMetricType } from '~/pages/modelServing/screens/metrics/ModelServingMetricsContext';
+import TrustyChart from '~/pages/modelServing/screens/metrics/TrustyChart';
+import { DomainCalculator } from '~/pages/modelServing/screens/metrics/types';
+
+const SPDChart = () => {
+  const DEFAULT_MAX_THRESHOLD = 0.1;
+  const DEFAULT_MIN_THRESHOLD = -0.1;
+
+  const domainCalc: DomainCalculator = (maxYValue) => ({
+    y: maxYValue > 0.1 ? [-1 * maxYValue - 0.1, maxYValue + 0.1] : [-0.2, 0.2],
```

Comment:
This is the functionality Rob asked for - he provided pseudo code of how he wanted the domain calculated which I converted to TS.

Reply:
I'd also say no, you could have a value that massively exceeds the threshold which could skew the graph. It makes sense to just add some padding around the max / min values being plotted.

Reply:
With respect to what Rob wants, don't make our code a magic number mess because you received a formula from someone else. 

`maxYValue > 0.1` seems like `maxYValue > DEFAULT_MAX_THRESHOLD` -- we should make it very deterministic and related so if things need to change it all changes with it. I'm guessing a lot, but there seems to be a common theme here with 0.1 and 0.2 differences, which I imagine are all part of a singular scale. 

All I was asking for was for this to be less "mystery" and more readable instead of a magic number kebab. If the numbers are not the same, write a description next to it. In my experience, this is how code becomes unmaintainable by anyone else but the developer who started it.

Reply:
Looking through this again, I see what you're saying, it clearly _is_ checking whether the value exceeds threshold, my bad - I missed that, I'll change it!

Reply:
After taking the time to grok what the formulas were actually doing I've tidied it up to this:
```
        y:
          maxYValue > DEFAULT_MAX_THRESHOLD
            ? [-1 * maxYValue - PADDING, maxYValue + PADDING]
            : [DEFAULT_MIN_THRESHOLD - PADDING, DEFAULT_MAX_THRESHOLD + PADDING],
```